setup-environment-internal

```diff
@@ -247,7 +247,7 @@ ln -sf "${MANIFESTS}" "${OEROOT}"/layers/
 
 DISTRO_DIRNAME=$(echo "${DISTRO}" | sed 's#[.-]#_#g')
 
-LMP_VER=$(git --git-dir ${MANIFESTS}/.git describe HEAD --tags --abbrev=0)
+LMP_VER="${LMP_VER-$(git --git-dir ${MANIFESTS}/.git describe HEAD --tags --abbrev=0)}"
```

Comment:
I would use `...${LMP_VER:-$(git`... ":-" - to handle pre-assigning not allowed empty value for `LMP_VER`

Reply:
but the `LMP_VER` can be empty and this is the default when we are not in the CI

Reply:
So we should handle empty LMP_VER in line 262 and don't add SSTATE_MIRRORS if LMP_VER is empty.

Reply:
No, when `LMP_VER` is empty we use git to retrieve the version. I will add you suggestion on the next push.
Thanks

Reply:
I'm sorry I wasn't clear in my last reply, you are right and I will add your suggestion to use `:-`  in this PR after we merge https://github.com/foundriesio/ci-scripts/pull/242